wrangler-docs/functions/null-handling.md

```diff
@@ -0,0 +1,125 @@
+# Null Handling Functions
+
+The collection of functions for handling nulls in your data.
+
+## IsNotNull
+Returns `true` when an expression does not evaluate to the `null` value.
+
+#### Namespace
+Global. No namespace qualifier required.
+
+#### Input
+any object or expression
+
+#### Output
+boolean (true/false)
+
+#### Example
+If `a` contains the `null`, `b` contains string `value` and `c` contains number `999` then
+If the input column does not contain a `null`, the output column contains the value of the input column.
+If the input column does contain a `null`, then the output column contains the string `null`. In the below
+examples, `col1` is `null`, `col2` is `value`, `col3` is `999` and `col4` is `null`.
+
+```
+set-column col1 IsNotNull(a) ? a : null
```

Comment:
we should replace this with a more useful example, as this is just a no-op. The example description is also confusing because of this. Use just a single expression like

set-column col1 IsNotNull(a) ? a : 0

Then explain what col1 evaluates to if a is null and if a is some value like 999.

Reply:
Replaced all methods with more understandable example.